Apps/Database/src/Delegates/DBUserPreferenceDelegate.cs

```diff
@@ -66,11 +61,11 @@ public DBResult<UserPreference> GetUserPreference(string hdid)
                 Payload = new UserPreference()
                 {
                     HdId = hdid,
-                    DismissedMyNotePopover = false,
+                    TutorialPopover = false,
                     Id = Guid.NewGuid(),
```

Comment:
There is an outstanding PR from Tiago with changes requested and this goes against those changes.